src/loaders/compute_tools/eggnog/eggnog.py

```diff
@@ -44,9 +43,13 @@ def _run_eggnog_single(
 
     # Save run info to a metadata file in the output directory for parsing later
     metadata = {'source_file': str(source_file),
-                'input_type': INPUT_TYPE}
-    with open(metadata_file, 'w') as f:
-        json.dump(metadata, f, indent=4)
+                'input_type': INPUT_TYPE,
+                'data_id': data_id,
+                'tool_name': 'eggnog',
+                'version': '2.1.12',
+                'command': command,
+                'reference_db': '5.0.2',}
```

Comment:
Both version and reference_db version seem like they could easly not be updated when they're updated elsewhere

Reply:
This comment applies to all the tools

Reply:
that's true. Grabbing them from the Dockerfile/versions file instead of manual input? I didn't build that logic because I think we are not changing versions very often. And I feel those metadata information will unlikely to be used at all. I don't see in the near future, anyone will run the same tool with different options and analyze the diffs.

Reply:
> Grabbing them from the Dockerfile/versions file instead of manual input?

that's what I was thinking, yeah. Is it possible for the versions file to get out of sync with the Dockerfile? Where are the version and refdb versions specified?

I think the minimum would be documenting the locations that the versions need to be changed and referring to that documentation in places where the version is likely to be changed

Reply:
👍 added reminder comments in docker file and versions.yaml

Reply:
:+1: